tests/MongoDB.Analyzer.Tests/MongoDB.Analyzer.Tests.csproj

```diff
@@ -28,6 +28,13 @@
     <ProjectReference Include="..\..\src\MongoDB.Analyzer\MongoDB.Analyzer.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />
+    <None Remove="MongoDB.Bson" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />
```

Comment:
Probably can revert all those changes in csproj file?

Reply:
Reverted